packages/dto/src/tables/users/user.rs

```diff
@@ -43,6 +43,9 @@ pub struct User {
     #[api_model(many_to_many = group_members, foreign_table_name = groups, foreign_primary_key = group_id, foreign_reference_key = user_id)]
     #[serde(default)]
     pub groups: Vec<Group>,
+    #[api_model(one_to_many = suggested_users, foreign_primary_key = user_id)]
```

Comment:
```suggestion
    #[api_model(one_to_many = suggested_users, foreign_key = user_id)]
```

one_to_may supports `foreign_key`
`many_to_many` supports `foreign_primary_key` and `foreign_reference_key` @blikq